opensrp/composed/sh/start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Import data
+
+/usr/local/bin/wait-for-it.sh mysql:3306
+
+mysql -u root -p"$MYSQL_ROOT_PASSWORD" -h mysql -e "CREATE DATABASE $MYSQL_MOTECH_DATABASE /*\!40100 DEFAULT CHARACTER SET utf8 */;"
```

Comment:
`-h mysql` could you use a variable for the mysql host as well aka `$MYSQL_HOST`, you could set the variable at the start of the script.